src/test/scala/org/broadinstitute/dsde/firecloud/MockServers.scala

```diff
@@ -136,6 +184,47 @@ object MockServers {
           )
           .withStatusCode(OK.intValue)
       )
+
+    // entity-level responses
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleValid.wsNamespace}/${mockSampleValid.wsName}/entities")
+          .withBody(mockSampleValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockPairValid.wsNamespace}/${mockPairValid.wsName}/entities")
+          .withBody(mockPairValid.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Created.intValue)
+      )
+
+    MockServers.workspaceServer
+      .when(
+        request()
+          .withMethod("POST")
+          .withPath(s"/workspaces/${mockSampleConflict.wsNamespace}/${mockSampleConflict.wsName}/entities")
+          .withBody(mockSampleConflict.toJson.compactPrint)
+          .withCookies(cookie)
+      ).respond(
+        response()
+          .withHeaders(header)
+          .withStatusCode(Conflict.intValue)
+      )
```

Comment:
Also a great idea - you could rely on that for mock functionality, and whoever provides the mock (for us presumably we would provide Rawls mock services) we would implement that however we see fit - which would ideally be using the existing code.  I like the idea of all of these mocks living in a common place along with the models